### PR TITLE
Remove package references from .Sources projects

### PR DESCRIPTION
**Pull request by kichalla**
@rynowak @davidfowl

### CODE REVIEW

File: src/Microsoft.Dnx.Compilation.CSharp.Common/project.json [hunk 1]
```diff
@@ -16,6 +23,7 @@
         },
         "dnxcore50": {
             "dependencies": {
+                "System.Console": "4.0.0-beta-*",
```
Comment (rynowak) [inline src/Microsoft.Dnx.Compilation.CSharp.Common/project.json]: This seems like an odd dep for this package - what's it used for?

File: src/Microsoft.Framework.CommandLineUtils.Sources/project.json [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
     "shared": "**/*.cs",
-    "frameworks" : {
-        "dnx451" : { },
-        "dnxcore50" : { 
+    "frameworks": {
+        "dnx451": { },
+        "dnxcore50": {
             "dependencies": {
-                "System.Collections": "4.0.11-beta-*",
                 "System.Runtime": "4.0.21-beta-*"
```
Comment (rynowak) [inline src/Microsoft.Framework.CommandLineUtils.Sources/project.json]: I expected this to just be empty. What happens without `System.Runtime`?

File: src/Microsoft.Dnx.Runtime.Sources/project.json [hunk 3]
```diff
@@ -8,16 +11,7 @@
...
-                "System.ComponentModel": "4.0.1-beta-*",
-                "System.Console": "4.0.0-beta-*",
-                "System.Diagnostics.Debug": "4.0.11-beta-*",
-                "System.Linq": "4.0.1-beta-*",
-                "System.Reflection": "4.0.11-beta-*",
-                "System.Runtime": "4.0.21-beta-*",
-                "System.Runtime.Extensions": "4.0.11-beta-*",
-                "System.Threading": "4.0.11-beta-*",
-                "System.Threading.Tasks": "4.0.11-beta-*"
+                "System.Runtime": "4.0.21-beta-*"
```
Comment (rynowak) [inline src/Microsoft.Dnx.Runtime.Sources/project.json]: I expected this to just be empty. What happens without `System.Runtime`?

Comment (davidfowl) [inline src/Microsoft.Dnx.Runtime.Sources/project.json]: You need it. These packages still end up turning into dlls (unfortunately)

Comment (rynowak) [inline src/Microsoft.Dnx.Runtime.Sources/project.json]: :crying_cat_face:

File: src/Microsoft.Dnx.Compilation.CSharp.Common/project.json (hunk 1 above)
Comment (davidfowl) [inline src/Microsoft.Dnx.Compilation.CSharp.Common/project.json]: Logging

File: src/Microsoft.Dnx.Runtime.Sources/project.json (hunk 3 above)
Comment (dougbu) [inline src/Microsoft.Dnx.Runtime.Sources/project.json]: @davidfowl VS also seems to _compile_ `.Sources` projects by default. Does that imply we need to keep some of these dependencies? Or should @kichalla disable building the projects in the `.sln` files (here `DNX.sln`)?

Comment (davidfowl) [inline src/Microsoft.Dnx.Runtime.Sources/project.json]: @kichalla shouldn't disable anything. Lets keep this change isolated.

Comment (rynowak) [inline src/Microsoft.Dnx.Runtime.Sources/project.json]: He's not talking about the assembly-in-sources-package issue.

VS will show you build errors unless you tell it not to build the project in the solution config.

File: src/Microsoft.Dnx.Compilation.CSharp.Common/project.json (hunk 1 above)
Comment (kichalla) [inline src/Microsoft.Dnx.Compilation.CSharp.Common/project.json]: Yes, it for Logger which is part of `Dnx.Runtime.Sources`(which this project references)

File: src/Microsoft.Framework.CommandLineUtils.Sources/project.json (hunk 2 above)
Comment (kichalla) [inline src/Microsoft.Framework.CommandLineUtils.Sources/project.json]: I get a build error saying `error CS8021: No value for RuntimeMetadataVersion found. No assembly containing System.Object was found nor was a value for RuntimeMetadataVersion specified through options.`

**Comment by kichalla** [comment]
@rynowak @davidfowl any more comments?

**Comment by davidfowl** [comment]
:shipit:

**Comment by kichalla** [comment]
1728b3aa4281aac968310e846ca8fe94034869cc

**Pull request closed without merging**